docs/tutorials/platform_beginner/3_customizing_the_general_layout.md

```diff
@@ -37,25 +37,25 @@ This tells Platform to use the `template` when rendering any content referenced
 
     To clear the cache:
 
-    ``` bash
+    ​``` bash
     $ php bin/console cache:clear
-    ```
+    ​```
 
 ## Creating the template
 
 1. [Download index.html](https://raw.githubusercontent.com/bdunogier/platform-workshop/master/src/Workshop/TutorialBundle/Resources/public/index.html)
-1. Save it in `app/Resources/views` as `app/Resources/views/full/root_folder.html.twig`.
-1. Refresh the site's root and you should see the site's structure, but without any styles or images. Let's fix this.
-1. Edit the `root_folder.html.twig` template.
+2. Save it in `app/Resources/views` as `app/Resources/views/full/root_folder.html.twig`.
```

Comment:
There's no need for this change, markdown automatically changes the 1s into proper numbers.